src/main/java/org/folio/bulkops/service/NoteTableUpdater.java

```diff
@@ -146,15 +155,17 @@ public void extendHeadersWithNoteTypeNames(int notesInitialPosition, List<Cell>
     headers.addAll(notesInitialPosition, cellsToInsert);
   }
 
-  public List<String> enrichWithNotesByType(List<String> list, int notesPosition, List<String> noteTypeNames, List<TenantNotePair> tenantNotePairs) {
+  public List<String> enrichWithNotesByType(List<String> list, int notesPosition, List<String> noteTypeNames,
+                                            List<TenantNotePair> tenantNotePairs, boolean forInstances) {
```

Comment:
Hi @obozhko-folio , you could use here the number of fields as argument instead of forInstances flag.

Reply:
I think it's clearer to use flag to divide instances and holdings/items.